p4-16/spec/P4-16-spec.mdk

```diff
@@ -7090,7 +7094,9 @@ error {
     NoMatch,           /// 'select' expression has no matches.
     StackOutOfBounds,  /// Reference to invalid element of a header stack.
     HeaderTooShort,    /// Extracting too many bits into a varbit field.
-    ParserTimeout      /// Parser execution time limit exceeded.
+    ParserTimeout,     /// Parser execution time limit exceeded.
+    ParserInvalidAlignment  /// extract or advance would leave parser with
```

Comment:
This error name is very specific. How about InvalidArgument, which could probably be reused for other purposes?

Reply:
done